### PR TITLE
ES6 Const Let Nathan Chon

### PR DESCRIPTION
**Pull request by Nathan-Chon**
image

### CODE REVIEW

File: es6-const-let/index.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>es6-const-let</title>
+</head>
+<body>
+  <script>
+    var greetings = [
```
Comment (Zhiwutian) [inline es6-const-let/index.html:11]: this one needs to be updated

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
one small update needed!

**Comment by Nathan-Chon** [comment]
image

Revised version fixed greeting variable

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**